<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checker</class>
 <widget class="QMainWindow" name="Checker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>680</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Checker</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #A0AECD;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelSettings">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Проверка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LENameFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>661</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid black;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid yellow;
}

QLineEdit::placeholder {
  color: #767e89;
}</string>
    </property>
    <property name="text">
     <string>Файлы не выбраны</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>600</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEErrorLine">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid yellow;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid yellow;
}

QLineEdit::placeholder {
  color: #767e89;
}</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Помощь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearFiles">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Отчистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenRes">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: solid black;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid black;
  outline: 0px;
}
QPushButton:hover {
  background-color: #6E6E6E;
  border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Открыть в проводнике</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEResLine">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid yellow;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid yellow;
}

QLineEdit::placeholder {
  color: #767e89;
}</string>
    </property>
    <property name="text">
     <string>Проверено N файлов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
